src/OpenSpaceToolkit/IO/IP/TCP/HTTP/Client.cpp

```diff
@@ -227,7 +232,7 @@ File Client::Fetch(const URL& aUrl, const Directory& aDirectory)
         curl_easy_setopt(curlPtr, CURLOPT_WRITEFUNCTION, writeDataFunction);
 
         curl_easy_setopt(curlPtr, CURLOPT_FAILONERROR, true);
-        // curl_easy_setopt(curlPtr, CURLOPT_FOLLOWLOCATION, 1L) ;
+        curl_easy_setopt(curlPtr, CURLOPT_FOLLOWLOCATION, aFollowCount);
```

Comment:
@kyle-cochran I honestly can't remember why this was commented out. If it is covered with tests and passes, I think we should be good